Source/JavaScriptCore/builtins/TypedArrayPrototype.js

```diff
@@ -197,17 +197,23 @@ function typedArrayMergeSort(array, valueCount, comparator)
     var buffer = new constructor(valueCount);
     var dst = buffer;
     var src = array;
+    if (@isResizableOrGrowableSharedTypedArrayView(array)) {
+        src = new constructor(valueCount);
+        for (var i = 0; i < valueCount; ++i)
```

Comment:
Maybe could use `constructor.prototype.set` here if it's unchanged from the original typed array's prototype?

Reply:
Yeah, if the array is backed by growable SharedArrayBuffer, then, it is possible that the size can be changed at any time. So we cannot copy valueCount elements correctly. We could improve performance, but this path is already a slow path, so maybe, slow performance is OK for now. (if comparator is not passed, we use C++'s super fast sort path, and this path is used for SharedArrayBuffer backing case etc.).